### DISCUSSION TITLE
Installing

### DISCUSSION THREAD

**Discussion by PaulWilson859** [sebr/bhyve-home-assistant#129] [General]
I have been trying to figure out how to install Bhyve.  I have downloaded it through HACS but when i got to install in home assitant nothing shows up.

**Comment by sebr** [comment]
Please see the readme: https://github.com/sebr/bhyve-home-assistant#configuration

**Reply by petereit** [reply]
All that link says is, "This integration should be configured from the Home Assistant devices & services UI page."

That's a little vague.

**Reply by sebr** [reply]
updated with latest 3.0.0 release

**Reply by heffneil** [reply]
I go to integrations and click the + Add Integration (in Home Assistant) and Orbit doesn't show up?  HACS shows it installed now so I am unclear what I am missing...

**Reply by petereit** [reply]
Pretty sure you need to add some YAML to your configuration file. The documentation is pretty vague there.

**Comment by heffneil** [comment]
When I added bhyve: to my configuration.yaml it told me that the integration couldn't be found.

**Comment by heffneil** [comment]
I was running 2022.7.1 I upgraded to 2022.7.5 and it appears to be working now-  thanks!

**Comment by TerryDes** [comment]
I am trying to add the BHyve app and have followed the instructions but when I try to add the integration it can't be found. Is there anything that needs to be added to the config file?

**Reply by sebr** [reply]
Nothing is needed in the config file if you are using version 3.x or later. Please make sure you haven't missed any of the [installation steps](https://github.com/sebr/bhyve-home-assistant#1-install-via-hacs) - including restarting HA!

**Reply by TerryDes** [reply]
Ok, well I have restarted 5 or six times. Going to try restarting the VM.

**Reply by TerryDes** [reply]
Ok, got it to go. I had cleared the cache in my browser a few times but all I had to do was to use a different browser and it worked. 

Thank you for the quick assistance!

**Comment by heffneil** [comment]
I did not add anything to the config file.  Just installed and restarted and then logged in under integrations.

**Comment by heffneil** [comment]
Which HA version are you running?  Might want to update to latest?

**Comment by DavidLovings** [comment]
Help:

I installed through HACS, Restarted HA, even restarted VM but when I go to Devices/Services and Add Integration I see Orbit B-Hyve but get the following message:

Error
Config flow could not be loaded: {"message":"Invalid handler specified"}

I did try a different browser for HA...but no go...

**Reply by sebr** [reply]
Which version of HA are you running?

**Reply by DavidLovings** [reply]
2023.3.6

**Reply by DavidLovings** [reply]
I see bhyve in custom_components

**Reply by DavidLovings** [reply]
For whatever reason, I reinstalled and it worked this time.

**Reply by DavidLovings** [reply]
It took about 5 times.  Maybe it did not finish the download the prior times.

**Reply by DavidLovings** [reply]
Thank You
